tox.ini

```diff
@@ -34,7 +34,7 @@ deps =
     jupyter
     jupytext
 commands =
-    jupytext --execute docs/tutorial_notebooks/*.ipynb
+    jupytext --execute docs/tutorial_notebooks/*.md
```

Comment:
that was accidentally reverted when I resolved a merge conflict earlier today. `.md` is the correct one.